core/chains/evm/client/errors.go

```diff
@@ -125,7 +127,7 @@ var arbitrum = ClientErrors{
 	// https://app.shortcut.com/chainlinklabs/story/16801/add-full-support-for-incorrect-nonce-on-arbitrum
 	NonceTooLow: regexp.MustCompile(`(: |^)invalid transaction nonce$|(: |^)nonce too low(:|$)`),
 	// TODO: Is it terminally or replacement?
-	TerminallyUnderpriced: regexp.MustCompile(`(: |^)gas price too low$`),
```

Comment:
@samsondav is this the correct mapping?

Reply:
No, I don't think so. This is an EIP1559 error... do we need to implement EIP1559 support for arbitrum?

Reply:
Surely we must be able to handle this error even if we are sending legacy transactions?
Which is the correct mapping, if not `TerminallyUnderpriced`?